tests/unit/conftest.py

```diff
@@ -54,9 +55,7 @@ def pyramid_request(db_session):
     pyramid_request.feature = mock.create_autospec(
         lambda feature: False, return_value=False  # pragma: no cover
     )
-    pyramid_request.lti_user = LTIUser(
-        "TEST_USER_ID", "TEST_OAUTH_CONSUMER_KEY", "TEST_ROLES"
-    )
+    pyramid_request.lti_user = factories.LTIUser()
```

Comment:
`pyramid_request.lti_user` is now a different randomly-generated `LTIUser` each time, so tests can no longer invisibly depend on `"TEST_USER_ID"`, "TEST_OAUTH_CONSUMER_KEY"` and `"TEST_ROLES"` having been passed to it